lib/bundle.js

```diff
@@ -220,6 +220,13 @@ function scopeCSS(path, blockContent, $) {
   return styles;
 }
 
+const validStyle = (val) => {
```

Comment:
Just realized we used named functions in this codebase. So this needs to be:

`function validStyle(val) {`